app/controllers/bookings_controller.rb

```diff
@@ -17,4 +35,11 @@ def edit
 
   def destroy
   end
-end
+
+  private
+
+  def booking_params
+    params.require(:booking).permit(:date_start, :date_end, :user_id, :costume_id)
```

Comment:
Same comment as above, user is and costume id is not needed here if not entered in the form